Dockerfile

```diff
@@ -1,37 +1,39 @@
-FROM phusion/baseimage:0.9.19
+FROM phusion/baseimage:0.9.22
```

Comment:
I'm using `python:3.6.1-slim` in one of my docker images, but just raw `ubuntu:16.04` in another. You can see the differences between the two [here](https://github.com/docker-library/python/blob/d3c5f47b788adb96e69477dadfb0baca1d97f764/3.6/jessie/slim/Dockerfile). (Note that the python-slim image is based off of Debian instead of Ubuntu)